test/unchain-variables-test.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable one-var, prefer-const */
+var foo = true,
+  // A comment
+  bar = false;
+const baz = 1,
+  fiz = '2';
+let buzz = 3.3,
+  biz = {};
+var hello, hi;
```

Comment:
can you add two tests that show:

* this works for more than 2 variables
* It works both for declarations where some variables are being initialized to a value and some aren't.

Your codemod covers this, would just be nice to express that in the tests.